src/c++/perf_analyzer/genai-perf/genai_perf/llm_metrics.py

```diff
@@ -182,11 +186,18 @@ def _preprocess_data(self, data: List, attr: str) -> List[Union[int, float]]:
 
     def _calculate_mean(self, data: List[Union[int, float]], attr: str) -> None:
         avg = np.mean(data)
+        self._stats_dict[attr]["avg"] = str(avg)
         setattr(self, "avg_" + attr, avg)
 
     def _calculate_percentiles(self, data: List[Union[int, float]], attr: str) -> None:
         p25, p50, p75 = np.percentile(data, [25, 50, 75])
         p90, p95, p99 = np.percentile(data, [90, 95, 99])
+        self._stats_dict[attr]["p99"] = str(p99)
+        self._stats_dict[attr]["p95"] = str(p95)
+        self._stats_dict[attr]["p90"] = str(p90)
+        self._stats_dict[attr]["p75"] = str(p75)
+        self._stats_dict[attr]["p50"] = str(p50)
+        self._stats_dict[attr]["p25"] = str(p25)
```

Comment:
I think the numbers should stay as numbers if these are going to be used in other places than exporting to json file.

Reply:
Updated them to floats